docs/install/from_source.rst

```diff
@@ -94,7 +94,7 @@ The configuration of TVM can be modified by `config.cmake`.
 
   - LLVM 4.0 or higher is needed for build with LLVM. Note that version of LLVM from default apt may lower than 4.0.
   - Since LLVM takes long time to build from source, you can download pre-built version of LLVM from
-    `LLVM Download Page <http://releases.llvm.org/download.html>`_.
+    `LLVM Download Page <https://github.com/llvm/llvm-project/releases/>`_.
```

Comment:
The previous page is a little bit more direct on what you need to download for a given platform. The GitHub Releases will show a list of ~50 files that may or may not be clear. I suggest adding a bit more clarity on what the user needs to download in this case.

My understanding is that you need "clang+llvm-<version>-<platform>.tar.xz", unless you're on Windows, that you'd be looking for "LLVM-<ver>-win<32 or 64>.exe".